_data/integrations.yml

```diff
@@ -0,0 +1,626 @@
+---
+#- id: [lowercase, words separated by hyphen]
+#  name:
+#  url:
+#  type: [ tms, cat ]
+#  platform: [ cloud-based or desktop ]
```

Comment:
Here it's not super clear that this one is not a list but the one above is.

I'd probably use `|` instead of `or` here.

Reply:
I'd go with a schema more like this:

```
# - id: <string> # slug
#   name: <string>
#   url: <string> # link
#   types: [ 'tms' | 'cat', ... ] # Functionality that it has
#   platform: 'cloud-based' | 'desktop'
#   open_source: true | false
#   engine_integrations:
#      - service: <string> # slug of the corresponding engine id
...
```